docs/src/main/asciidoc/getting-started-knative.adoc

```diff
@@ -8,10 +8,9 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 include::./attributes.adoc[]
 :experimental:
 
-
 This guide covers:
 
-* The deployment of the application to Kubernetes
+* The deployment of the application to Knative
```

Comment:
Maybe let's remove the bullet list if you have only one item (and add a dot at the end of the sentence)?